### PR TITLE
Add -branch flag

### LINKED ISSUE #1: Add a 'branch' information
Knowing the branch on which the bench were run could be interesting.
It may help deciding if a feature branch can be merge for example.
It can also help to see the evolution of each branch over time.

Not knowing well influxDB, I don't know if it should be a tag or a field.

### PR DESCRIPTION
**Pull request by SVilgelm**
Fix #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +60,7 @@ If you set it to an empty string, `grade` will print line protocol to stdout.
 * The Go version subcommand will produce a string like `go1.6.2 darwin/amd64`, but you can use any string you'd like.
 * The revision subcommand is the full SHA of the commit, but feel free to use a git tag name or any other string.
 * The timestamp is a Unix epoch timestamp in seconds.
+* The branch subcommand is the name of the current branch.
```
Comment (mark-rushakoff) [inline README.md]: Please clarify that the branch is optional.

File: cmd/grade/main.go [hunk 2]
```diff
@@ -29,6 +29,7 @@ func init() {
 	flag.Int64Var(&unixTime, "timestamp", 0, "Unix epoch timestamp (in seconds) to apply when storing all benchmark results")
 	flag.StringVar(&cfg.Revision, "revision", "", "Revision of the repository used to generate benchmark results")
 	flag.StringVar(&cfg.HardwareID, "hardwareid", "", "User-specified string to represent the hardware on which the benchmarks were run")
+	flag.StringVar(&cfg.Branch, "branch", "", "Branch of the repository used to generate benchmark results")
```
Comment (mark-rushakoff) [inline cmd/grade/main.go]: Please clarify that the branch is optional.

File: grade.go [hunk 3]
```diff
@@ -32,6 +32,9 @@ type Config struct {
 
 	// HardwareID is a user-specified string to represent the hardware on which the benchmarks have run.
 	HardwareID string
+
+	// Branch is the tag value to use to indicate which branch of the repository was used for the benchmarks that have run.
```
Comment (mark-rushakoff) [inline grade.go:36]: The godoc should also clarify that the branch is optional.

**Review by mark-rushakoff — CHANGES_REQUESTED** [review]
Thanks, the code change looks good. Can you please update the documentation to clarify that branch is optional?

**Comment by mark-rushakoff** [comment]
And we will need you to sign the CLA as well, thanks. https://influxdb.com/community/cla.html

File: README.md (hunk 1 above)
Comment (SVilgelm) [reply README.md]: done

File: cmd/grade/main.go (hunk 2 above)
Comment (SVilgelm) [reply cmd/grade/main.go]: done

File: grade.go (hunk 3 above)
Comment (SVilgelm) [reply grade.go:36]: done

**Comment by SVilgelm** [comment]
> And we will need you to sign the CLA as well, thanks. https://influxdb.com/community/cla.html

Done

**Review by mark-rushakoff — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**